smoketest/scripts/cli/test_protocols_segment-routing.py

```diff
@@ -47,14 +48,64 @@ def tearDown(self):
     def test_srv6(self):
         interfaces = Section.interfaces('ethernet', vlan=False)
         locators = {
-            'foo' : { 'prefix' : '2001:a::/64' },
-            'foo' : { 'prefix' : '2001:b::/64', 'usid' : {} },
+            'foo': {'prefix': '2001:a::/64'},
+            'foo1': {'prefix': '2001:b::/64', 'usid': {}},
+            'foo2': {'prefix': '2001:c::/64', 'format': 'uncompressed-f4024'},
+            'foo3': {
+                'prefix': '2001:d::/48',
+                'block-len': '32',
+                'node-len': '16',
+                'func-bits': '16',
+                'usid': {},
+                'format': 'usid-f3216',
+            },
         }
 
         for locator, locator_config in locators.items():
-            self.cli_set(base_path + ['srv6', 'locator', locator, 'prefix', locator_config['prefix']])
+            self.cli_set(
+                base_path
+                + ['srv6', 'locator', locator, 'prefix', locator_config['prefix']]
+            )
+            if 'block-len' in locator_config:
+                self.cli_set(
+                    base_path
```

Comment:
I see no ruff rule making this change, was it done by IDE?

Reply:
Running ruff manually I only see:

```
(13:10) vyos_bld 5fcab0ce5eb2:/vyos/vyos-1x [current] # ruff check --config ruff.toml smoketest/scripts/cli/test_protocols_segment-routing.py
smoketest/scripts/cli/test_protocols_segment-routing.py:51:13: F601 Dictionary key literal `'foo'` repeated
   |
49 |         locators = {
50 |             'foo' : { 'prefix' : '2001:a::/64' },
51 |             'foo' : { 'prefix' : '2001:b::/64', 'usid' : {} },
   |             ^^^^^ F601
52 |         }
   |
   = help: Remove repeated key literal `'foo'`

smoketest/scripts/cli/test_protocols_segment-routing.py:71:39: F541 [*] f-string without any placeholders
   |
69 |             self.assertEqual(sysctl_read(f'net.ipv6.conf.{interface}.seg6_require_hmac'), '0') # default
70 |
71 |         frrconfig = self.getFRRconfig(f'segment-routing', daemon=zebra_daemon)
   |                                       ^^^^^^^^^^^^^^^^^^ F541
72 |         self.assertIn(f'segment-routing', frrconfig)
73 |         self.assertIn(f' srv6', frrconfig)
   |
   = help: Remove extraneous `f` prefix

smoketest/scripts/cli/test_protocols_segment-routing.py:72:23: F541 [*] f-string without any placeholders
   |
71 |         frrconfig = self.getFRRconfig(f'segment-routing', daemon=zebra_daemon)
72 |         self.assertIn(f'segment-routing', frrconfig)
   |                       ^^^^^^^^^^^^^^^^^^ F541
73 |         self.assertIn(f' srv6', frrconfig)
74 |         self.assertIn(f'  locators', frrconfig)
   |
   = help: Remove extraneous `f` prefix

smoketest/scripts/cli/test_protocols_segment-routing.py:73:23: F541 [*] f-string without any placeholders
   |
71 |         frrconfig = self.getFRRconfig(f'segment-routing', daemon=zebra_daemon)
72 |         self.assertIn(f'segment-routing', frrconfig)
73 |         self.assertIn(f' srv6', frrconfig)
   |                       ^^^^^^^^ F541
74 |         self.assertIn(f'  locators', frrconfig)
75 |         for locator, locator_config in locators.items():
   |
   = help: Remove extraneous `f` prefix

smoketest/scripts/cli/test_protocols_segment-routing.py:74:23: F541 [*] f-string without any placeholders
   |
72 |         self.assertIn(f'segment-routing', frrconfig)
73 |         self.assertIn(f' srv6', frrconfig)
74 |         self.assertIn(f'  locators', frrconfig)
   |                       ^^^^^^^^^^^^^ F541
75 |         for locator, locator_config in locators.items():
76 |             self.assertIn(f'   locator {locator}', frrconfig)
   |
   = help: Remove extraneous `f` prefix

Found 5 errors.
[*] 4 fixable with the `--fix` option.
```

Reply:
Try `ruff format --config ruff.toml smoketest/scripts/cli/test_protocols_segment-routing.py`.

It was requiring changes before I modified this test to include the `format` option.